README.md

```diff
@@ -112,6 +112,7 @@ A curated list of awesome things related to <a href='https://ui.shadcn.com/' tar
 - [simplekit](https://github.com/vaunblu/SimpleKit) - Responsive connect wallet and account component built on top of Wagmi and shadcn/ui.
 - [sortable](https://github.com/sadmann7/sortable) - A sortable component built with shadcn/ui, radix ui, and dnd-kit.
 - [stocks](https://github.com/aryanvichare/stocks) - Stock Picker using Next.js, React Server Components, and shadcn/ui charts.
+- [stunning-ui](stunningui.design) - Stunning UI is a collection of interactive Tailwind CSS components built for Vue and Nuxt.
```

Comment:
supposed to add `https://` or else github will consider this as a file name and redirect u to the file location 0-0

Reply:
okay i forgot :v